data/json/recipes/recipe_others.json

```diff
@@ -2424,6 +2424,66 @@
     "tools": [ [ [ "crucible", -1 ], [ "crucible_clay", -1 ] ] ],
     "components": [ [ [ "2x4", 3 ] ] ]
   },
+  {
+    "type": "recipe",
+    "result": "hammer_sledge_short",
+    "id_suffix": "forged",
+    "category": "CC_OTHER",
+    "subcategory": "CSC_OTHER_TOOLS",
+    "skill_used": "fabrication",
+    "difficulty": 6,
+    "time": "7 h 40 m",
+    "autolearn": true,
```

Comment:
```suggestion
```
We're trying to reduce the number of autolearned recipes, and ones that require knowledge of blacksmithing definitely fall under this. I realize this puts them into a different spot than existing recipes, but at least it saves a bit of work later.

Reply:
Fair enough, I'm fine with that being removed. I only had autolearn there because the regular sledge had the field already and I largely copied from it with some slight changes. If you'd like, I can also remove it from the original sledge to help that goal along as this is already dealing with refactoring it.

Reply:
I'd appreciate that! Going forward we're going to want at least one, possibly a few (eg a beginner and a general) specific blacksmithing books with these recipes in them, and others. That's out of the scope of this PR but it's something you could consider for a followup if you like.